docs/content/release-notes-template.md

```diff
@@ -0,0 +1,35 @@
+## Spin <version>
+
+The <version> release of Spin brings a number of features, improvements and bug fixes.
+
+Some highlights in <version> at a glance:
+- <new feature linked to PR>
+- <new feature linked to PR>
+
+<List notable fixes, deprecations, breaking changes, etc.>
```

Comment:
Any tips to help the release engineer consolidate commits or a reference to a good summary made in previous releases might be useful and also keep the releases consistent?

Reply:
I added the following text under the release notes section: https://github.com/fermyon/spin/pull/2622/files#diff-c81ca86152e7d9db4536d4759d7ce52c22425cb618106f39ae65f1f2fd22e349R84-R86 What do you think?

Reply:
Awesome, that helps. Thanks for adding